From d4b0f895f9d621e43027cc3fa2c8d521128e418a Mon Sep 17 00:00:00 2001
From: Ahmed Ismail <Ahmed.Ismail@arm.com>
Date: Wed, 27 Dec 2023 13:35:16 +0000
Subject: [PATCH] platform-drivers: Remove RTE_components header file

RTE_Components header file is being included for both Corstone-300
and Corstone-310 platforms. However in the case of Corstone-300
platofrm, this file is not available anymore in the latest version of
cmsis-packs Hence, this header file inclusion is removed in the new
patch file.

Signed-off-by: Ahmed Ismail <Ahmed.Ismail@arm.com>
---
 source/hal/source/platform/mps3/include/platform_drivers.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/source/hal/source/platform/mps3/include/platform_drivers.h b/source/hal/source/platform/mps3/include/platform_drivers.h
index fc7dcc5..6a017bb 100644
--- a/source/hal/source/platform/mps3/include/platform_drivers.h
+++ b/source/hal/source/platform/mps3/include/platform_drivers.h
@@ -21,7 +21,6 @@
 #include "log_macros.h"     /* Logging related helpers. */
 
 /* Platform components */
-#include "RTE_Components.h" /* For CPU related defintiions */
 #include "timer_mps3.h"     /* Timer functions. */
 #include "user_input.h"     /* User input function */
 #include "lcd_img.h"        /* LCD functions. */
-- 
2.34.1

